ibm/service/power/ibm_pi_constants.go

```diff
@@ -283,6 +284,9 @@ const (
 	Attr_RemoteCopyID                                = "remote_copy_id"
 	Attr_RemoteCopyRelationshipNames                 = "remote_copy_relationship_names"
 	Attr_RemoteCopyRelationships                     = "remote_copy_relationships"
+	Attr_StoragePoolMapRemotePool                    = "remote_pool"
+	Attr_StoragePoolMapVolumePool                    = "volume_pool"
+	Attr_ReplicationPoolMap                          = "replication_pool_map"
```

Comment:
I think these are also sorted incorrectly

Reply:
Those variables were not meant for this PR and have been removed.